package test;

public class Exception {

	public static void main(String[] args) {
		// TODO Auto-generated method stub
		
		int i  = 9 ; 
		
		try {
			System.out.println();
		} catch (java.lang.Exception e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		}

	}

}
